content/posts/soyokaze/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "soyokaze!"
+img: "soyokaze-logo.png"
+date: 2021-04-20
+summary: "An osu! ruleset mimicking Genshin Impact&rsquo;s Ballads of Breeze mini-game."
```

Comment:
```suggestion
summary: "An osu! ruleset mimicking Genshin Impact's Ballads of Breeze mini-game."
```
I am not sure what happened here but I think this is wrong

Reply:
It's HTML numberic code for curly quote.